.eslintrc.js

```diff
@@ -68,6 +68,7 @@ module.exports = {
       },
       rules: {
         'jest/expect-expect': 0,
+        'jest/valid-expect': 0,
```

Comment:
Why is this change required globally? We can inline the rule disable comment.

Reply:
this is only scoped to the cypress tests, the assertion is not a jest one but the cypress one